easysid.cpp

```diff
@@ -152,42 +152,58 @@ EXPORT void set_filter_res(uint8_t res)
     sid.write(23, reg);
 }
 
-EXPORT void set_filter_enable(uint8_t voice)
+EXPORT void set_filter(uint8_t voices, bool enable)
```

Comment:
Don't use a bit mask for this. There's no reason to complicate the interface that way. If the caller needs to enable the filter on more than one voice, then call this function more than once.